### PR TITLE
updated sidebar handlebars

### PR DESCRIPTION
**Pull request by pascalcalixte26**
Please review my updates to sidebar handlebars.  I used body sidebar handlebar as template.  I didn't touch nav sidebar handlebar.

### CODE REVIEW

File: views/sidebars/carousel.handlebars [hunk 1]
```diff
@@ -1,19 +1,52 @@
 <a href="#" class="backButton button btn-secondary"><i class="fa fa-back"></i> Back</a>
 
-<h5>Image 1</h5>
-
-<form method="PUT" action="/api/customization/carousel" class="customizationForm">
+<form  method="PUT" action="/api/customization/carousel/{{index}}" class="customizationForm {{data.option}}">
```
Comment (michaelrosario) [inline views/sidebars/carousel.handlebars]: The customization for carousel is not an array, you can remove {{index}}

File: views/sidebars/carousel.handlebars [hunk 2]
```diff
@@ -1,19 +1,52 @@
...
 
-<input name="image-one" class="form-control mb-3"  value="" />
```
Comment (michaelrosario) [inline views/sidebars/carousel.handlebars:7]: We need to keep this.  It should look like this.
```<input name="image-one" class="form-control mb-3"  value="{{#if data.customization.image-one}}{{ data.customization.image-one}}{/if}" />```

File: views/sidebars/carousel.handlebars [hunk 3]
```diff
@@ -1,19 +1,52 @@
...
+<div class="item-one">
+<h5>Image 1</h5>
+<textarea name="columnOne" id="" class="form-control mb-3"  cols="18" rows="10" placeholder="image">
```
Comment (michaelrosario) [inline views/sidebars/carousel.handlebars]: See note above

File: views/sidebars/carousel.handlebars [hunk 4]
```diff
@@ -1,19 +1,52 @@
...
+{{#if data.customization.columnOne}}{{data.customization.columnOne}}{{/if}}
+</textarea>
+</div>
 
+<div class="item-two">
 <h5>Image 2</h5>
-<input name="image-two" class="form-control mb-3"  value="" />
```
Comment (michaelrosario) [inline views/sidebars/carousel.handlebars:10]: See note above

File: views/sidebars/carousel.handlebars [hunk 5]
```diff
@@ -1,19 +1,52 @@
...
+<textarea name="columnTwo" id="" class="form-control mb-3"  cols="18" rows="10" placeholder="image" >
+{{#if data.customization.columnTwo}}{{data.customization.columnTwo}}{{/if}}
+</textarea>
+</div>
 
+<div class="item-three">
 <h5>Image 3</h5>
-<input name="image-three" class="form-control mb-3"  value="" />
```
Comment (michaelrosario) [inline views/sidebars/carousel.handlebars:13]: See note above

File: views/sidebars/footer.handlebars [hunk 6]
```diff
@@ -1,19 +1,57 @@
...
-<H5>Instagram</H5>
+
 <form  method="PUT" action="/api/customization/footer" class="customizationForm">
-<input name="instagram" class="form-control mb-3" value="">
 
-<H5>Snapchat</H5>
-<input name="snapchat" class="form-control mb-3" value="" >
+<div class="item-one">
+<h5>Instagram</h5>
+<textarea name="columnOne" id="" class="form-control mb-3"  cols="18" rows="10" placeholder="instagram">
```
Comment (michaelrosario) [inline views/sidebars/footer.handlebars]: See note on previous file, we keep it as an input field and not a textarea.

**Review by michaelrosario — CHANGES_REQUESTED** [review]
We have to keep the input fields and names, the only thing we change what goes inside the `value=""`
Nice try nonetheless.

File: views/sidebars/carousel.handlebars (hunk 1 above)
Comment (pascalcalixte26) [reply views/sidebars/carousel.handlebars]: I removed index.  Having trouble with the original input code I overrode with the body code

File: views/sidebars/carousel.handlebars (hunk 2 above)
Comment (pascalcalixte26) [reply views/sidebars/carousel.handlebars:7]: Ok Updating now Thanks!

File: views/sidebars/carousel.handlebars [hunk 7]
```diff
@@ -1,34 +1,26 @@
...
+<form  method="PUT" action="/api/customization/carousel/" class="customizationForm {{data.option}}">
+
+    <div class="item-one">
+        <h5>Image 1</h5>
+        <input name="image-one" class="form-control mb-3"  value="{{#if data.carousel.customization.image-one}}{{ data.carousel.customization.image-one}}{/if}" />
+    </div>
+
+    <div class="item-two">
+        <h5>Image 2</h5>
+        <input name="image-two" class="form-control mb-3"  value="{{#if data.carousel.customization.image-one}}{{ data.carousel.customization.image-one}}{/if}" />
```
Comment (michaelrosario) [inline views/sidebars/carousel.handlebars]: this should be `{{data.carousel.customization.image-two}}`

File: views/sidebars/carousel.handlebars [hunk 8]
```diff
@@ -1,34 +1,26 @@
...
+    </div>
+
+    <div class="item-three">
+        <h5>Image 1</h5>
+        <input name="image-three" class="form-control mb-3"  value="{{#if data.carousel.customization.image-one}}{{ data.carousel.customization.image-one}}{/if}" />
```
Comment (michaelrosario) [inline views/sidebars/carousel.handlebars]: this is .image-three

**Review by michaelrosario — CHANGES_REQUESTED** [review]
Some more changes

**Pull request merged**